transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -514,4 +529,62 @@ void updateHelpText() {
         }
         helpText.getParent().layout();
     }
+
+    void updateManifestPackageImports(IProject project,
+                                      File pomFile) throws Exception {
+        IPath pomPath = project.getRawLocation() != null
+                        ? project.getRawLocation()
+                        : ResourcesPlugin.getWorkspace().getRoot().getLocation().append(project.getFullPath());
+        Model pomModel = MavenPlugin.getMaven().readModel(new File(pomPath.append("pom.xml").toOSString()));  //$NON-NLS-1$
+        Build build = pomModel.getBuild();
+        Map<String, Plugin> pluginsByName = build.getPluginsAsMap();
+        Plugin plugin = pluginsByName.get("org.apache.felix:maven-bundle-plugin"); //$NON-NLS-1$
+        if (plugin == null) {
+            plugin = new Plugin();
+            plugin.setGroupId("org.apache.felix"); //$NON-NLS-1$
+            plugin.setArtifactId("maven-bundle-plugin"); //$NON-NLS-1$
+            plugin.setVersion("3.2.0"); //$NON-NLS-1$
+            plugin.setExtensions(true);
+            build.addPlugin(plugin);
+        }
+        if (!(plugin.getConfiguration() instanceof Xpp3Dom)) throw new Exception(Messages.TransformationEditor_invalidPomConfiguration);
```

Comment:
Another side effect is that the new plugin is not persisted into the pom if the above occurs.